test/src/test/java/hudson/slaves/JNLPLauncherTest.java

```diff
@@ -69,7 +71,9 @@
 public class JNLPLauncherTest {
     @Rule public JenkinsRule j = new JenkinsRule();
     
-    @Rule public TemporaryFolder tmpDir = new TemporaryFolder(); 
+    @Rule public TemporaryFolder tmpDir = new TemporaryFolder();
+
+    @Rule public LoggerRule logging = new LoggerRule().record(Slave.class, Level.FINE);
```

Comment:
unused in test - is this to show that your fine logging does not blow up due to invalid parameters or other. (in which case a comment in the code would help future reviewers)

Reply:
Does not affect test pass status, just there to help developers understand what is happening when the test is being run interactively. I.e., the typical usage of `LoggerRule`.